fury/actor.py

```diff
@@ -3954,3 +3956,101 @@ def uncertainty_cone(
     angles = main_dir_uncertainty(evals, evecs, signal, sigma, b_matrix)
 
     return double_cone(centers, evecs, angles, colors, scales, opacity)
+
+
+def texture_on_cube(negx, negy, negz, posx, posy, posz, centers=(0, 0, 0)):
+    """Map RGB or RGBA textures on a cube
+
+    Parameters
+    ----------
+    negx : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
+    negy : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
+    negz : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
+    posx : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
+    posy : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
+    posz : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
```

Comment:
why it should be uint8?

Reply:
I tested other datatypes and the video is working ok. I added it because `actor.py/texture_update` had this constraint for `arr` parameter. Looks like it doesn't affect `viz_play_cube`. 

I tested different videos with uint16, uint8 and all of them worked without crashing. The only weird thing was uint16 causes a red screen for the first frame (and this is before `texture_update` gets even called inside `viz_play_cube` if I set dtype as uint16. But I do not think that should be taken care by FURY. `cv.read()` default return dtypes work well without errors)

Removed the constraint. Resolved.